scripts/hello_word_loracsilva.js

```diff
@@ -5,7 +5,6 @@
 
 'use strict'
 
-const hello = "Hello World"
```

Comment:
You should create a new file

Reply:
I created a new file "hello_world_memphis1983", should I do another?